### DISCUSSION TITLE
Does Sahi involve any training step?

### DISCUSSION THREAD

**Discussion by tbonewmy** [obss/sahi#609] [Q&A] [answered]
Hi there,
if I have a model A trained on a group of image B. say I want to use Sahi to improvement detection of smaller objects in B. When I use Sahi, does it continues to do extra training on those rescaled image path cutouts before it do inference?

**Comment by fcakyon** [comment, marked as answer]
Thanks for the question. SAHI is only for inference. You can directly use it with your fine-tuned model with no additional training. However we show in our paper that slicing aided fine-tuning increases inference accuracy: https://arxiv.org/abs/2202.06934

You can also check this repo for slicing-aided fine-tuning config files (for mmdetection framework)
